packages/react/alert-dialog/src/AlertDialog.tsx

```diff
@@ -62,6 +64,9 @@ const AlertDialogContent = React.forwardRef((props, forwardedRef) => {
             event.preventDefault();
             cancelRef.current?.focus({ preventScroll: true });
           })}
+          onPointerDownOutside={composeEventHandlers(contentProps.onPointerDownOutside, (event) =>
```

Comment:
I think we also need to prevent the escape keydown one.

Reply:
May be worth rejigging what's exposed from `Dialog` to facilitate the changes here also.

Reply:
Ha cool, good to know. I suppose that's still a meaningful user action, as opposed to clicking outside by mistake potentially.

Reply:
We probably should just hardcode it then and remove it from the types.

Reply:
> We probably should just hardcode it then and remove it from the types.

That's fine with me, are we sure consumers won't need access to the handler for a different purpose?